kedro-org/kedro-viz
# Fix font-size inconsistencies

Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Fixes #892.

## Development notes

I've updated the font sizes so they are correct and align with the designs. I also ran `npm install` and that's why there's a change to the `package-lock.json` file.

## QA notes

View the Gitpod deployed URL and check for any decimal places in the font sizes via the browser inspector.

Also note that not only was the font size of the elements in the node list wrong, the [height was wrong as well](https://zpl.io/br9JqPO). I fixed that and now the information density is greater, and to me looks better. See here...

**Before change on left | After the change on the right**

<img width="701" alt="Background" src="https://user-images.githubusercontent.com/16869061/173607554-4d89462e-5755-4db0-8ea9-2fee71cc6e18.png">

Let me know what you think!

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/908"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
